<?xml version="1.0"?>

<launch>

        <!--GREEN COLOUR SELECTION-->
        <param name="H_min_g" value="0"/>
        <param name="H_max_g" value="65"/>
        <param name="S_min_g" value="0"/>
        <param name="S_max_g" value="136"/>
        <param name="V_min_g" value="8"/>
        <param name="V_max_g" value="255"/>

    <node name="indicator_detection" pkg="vision" type="indicator_detection" output="screen">
        <!--TARJET SIZE-->
        <param name="teoric_size" value="0.64"/> <!--Value in m2-->
        <param name="altura_summit" value="0.392"/> <!--Value in m-->

        <!--TOPICS-->
        <param name="indicator_position_topic" value="/platform/indicator_position"/>        
        <param name="image_topic" value="/ardrone/bottom/image_raw"/>
        <param name="cam_info_topic" value="/ardrone/bottom/camera_info"/>
        <param name="altitude_topic" value="/altimeter"/>
        <param name="imu_topic" value="/ardrone/imu"/>
        <param name="cmd_topic_vel" value="/cmd_vel"/>
    </node>
 </launch>
